sdk/identity/identity/test/internal/identityClient.spec.ts

```diff
@@ -56,7 +56,11 @@ describe("IdentityClient", function() {
       mockHttp.tokenCredentialOptions
     );
     await assertRejects(credential.getToken("https://test/.default"), (error) => {
-      assert.equal(error.name, "CredentialUnavailableError");
+      if (isNode) {
+        assert.equal(error.name, "CredentialUnavailableError");
+      } else {
+        assert.equal(error.name, "AuthenticationError");
```

Comment:
Probably worth adding a comment

> Different from node because the node version uses msal.

Reply:
@HarshaNalluru what about, before the `if (isNode)`:

> // Keep in ind that this credential has different implementations in Node an in browsers.

Reply:
Anything, anywhere :)